bundler/lib/bundler/cli.rb

```diff
@@ -668,6 +670,8 @@ def inject(name, version)
       "If updating, prefer updating only to next minor version"
     method_option "major", :type => :boolean, :banner =>
       "If updating, prefer updating to next major version (default)"
+    method_option "pre", :type => :boolean, :banner =>
+      "If updating, allow consideration of prerelease gems"
```

Comment:
Just noticed that even though the PR mentions only `bundle update`, we're also adding this flag to `bundle lock`? That seems nice! Should we add a simple spec that runs `bundle lock --pre`? Just to make sure that flag is properly channeled through all the way to the version promoter.

Reply:
Good call! Added specs very similar to the ones for bundle update. I'll update the PR description